lib/openshift/machine_config_pool.rb

```diff
@@ -8,5 +8,10 @@ def spec(user: nil, cached: true, quiet: false)
       rr = raw_resource(user: user, cached: cached, quiet: quiet).dig("spec")
       MachineConfigPoolSpec.new rr
     end
+
+    def machineconfig(mc_name, user: nil, cached: true, quiet: false)
```

Comment:
I didn't want to request such approach in the other PR because machine config source is a complex object and we better have a special class for it. Let me have a look tomorrow.